models/marts/dim_customers.sql

```diff
@@ -0,0 +1,36 @@
+{{ config(materialized='table') }}
```

Comment:
you can drop this since you defined the materialization already in the `dbt_project.yml` file
```suggestion
```